content/5-New-Things.md

```diff
@@ -0,0 +1,129 @@
+streams in the area.
+
+**Major Flooding** - is defined to have extensive inundation of structures
+and roads. Significant evacuations of people and/or transfer of property
+to higher elevations are necessary. A **flood warning** should be issued
+if major flooding is expected during the event. In remote areas with few
+specific impacts, floods with 50-100 year recurrence interval would be
+assumed to be causing major flooding on streams in the area.
+
+<div class="grid-row">
+    <div class="grid-col-14 grid-offset-0">
+    {{< figure src="/static/5-new-things-pt1/image1.png" caption="A hydrograph during a flooding event displaying the flood stage benchmarks at Elm River at Westport, South Dakota. <https://waterdata.usgs.gov/monitoring-location/06471500/?agency_cd=USGS#parameterCode=00065&startDT=2020-07-10&endDT=2020-08-05>." alt="Screenshot of a hydrograph showing floodbenchmarks at Elm River at Westport, South Dakota" >}}
+    </div>
+</div>
+
+
+\#2 State of hydrograph for bookmarking and linking
+---------------------------------------------------
+
+The next-gen monitoring location pages serve a huge range of users from
+kayakers to emergency managers. Users are now able to bookmark and share
+a hydrograph in the way that they requested it. Users can share custom
+hydrographs generated with specific time-series statistics, parameter
+codes, dates, and compare settings. For example, a user may be
+interested in the changes in gage height at the [USGS gage Colorado
+River at Lee\'s Ferry,
+AZ](https://waterdata.usgs.gov/monitoring-location/09380000/) over the
+course of the November 2012 [Glen Canyon Dam High Flow Experimental
+Release](https://www.usbr.gov/uc/rm/gcdHFE/index.html). After choosing a
+custom time range and the gage height parameter, the [URL in the browser
+can be used to share or return to the same
+settings](https://waterdata.usgs.gov/monitoring-location/09380000/#parameterCode=00065&startDT=2012-11-01&endDT=2012-12-01).
+
+\#3 NLDI (The Hydro Network-Linked Data Index)
+----------------------------------------------
+
+By utilizing the Hydro Network-Linked Data Index
+([NLDI](https://waterdata.usgs.gov/blog/nldi-intro/)), monitoring
+locations can be put into the context of a catchment or basin. Each
+active stream site has an upstream basin in NLDI, as you can see here:
+
+<div class="grid-row">
+    <div class="grid-col-14 grid-offset-0">
+    {{< figure src="/static/5-new-things-pt1/image2.png" caption="Screenshot of the upstream basin for monitoring location05370000, [Yahara River at East Main Street at Madison, WI](https://waterdata.usgs.gov/monitoring-location/05428500/#parameterCode=00060)." alt="Screenshot of the upstream basin, shown as a grey overlay, for monitoring location05370000, [Yahara River at East Main Street at Madison, WI]" >}}
+    </div>
+</div>
+
+
+
+\#4 Brush & Zoom
+----------------
+
+The brush and zoom capability allows the user to easily select any part
```

Comment:
So technically we only have implemented "brush" in the D3 sense. The "zoom" feature involves  using the mouse or gestures on mobile to zoom in and out. We have not yet done this. I think you can leave this as is, but also wanted you to know that one of the things I think we should do is remove the brush from mobile and use zoom instead. We save real estated then and zoom is easier for user's to use on mobile.

Reply:
Oh, ok. I did not know the details there.